crates/nu-cli/src/commands/mkdir.rs

```diff
@@ -44,20 +44,20 @@ impl WholeStreamCommand for Mkdir {
     }
 }
 
-fn mkdir(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStream, ShellError> {
+async fn mkdir(args: CommandArgs, registry: &CommandRegistry) -> Result<OutputStream, ShellError> {
     let registry = registry.clone();
-    let stream = async_stream! {
-        let name = args.call_info.name_tag.clone();
-        let shell_manager = args.shell_manager.clone();
-        let (args, _) = args.process(&registry).await?;
-        let mut result = shell_manager.mkdir(args, name)?;
+    let name = args.call_info.name_tag.clone();
+    let shell_manager = args.shell_manager.clone();
+    let (args, _) = args.process(&registry).await?;
+    let mut result = shell_manager.mkdir(args, name)?;
```

Comment:
I think you might be able to just call `shell_manager.mkdir(args, name)` and let the result of it be the return value of the function

Reply:
Oh nice, yeah I totally missed that.  Yeah what I was doing here was totally unnecessary.